client/my-sites/github-deployments/deployments/index.tsx

```diff
@@ -46,11 +46,14 @@ export function GitHubDeployments() {
 		<PageShell
 			pageTitle={ __( 'GitHub Deployments' ) }
 			topRightButton={
-				<ConnectionWizardButton
-					onClick={ () => {
-						page( createDeploymentPage( siteSlug! ) );
-					} }
-				/>
+				deployments &&
```

Comment:
We don't need to check if `deployments` exists now that we're using optional chaining right next to it.